README.md

```diff
@@ -143,9 +143,9 @@ _Before you begin, make sure you are running Python 3.7/3.8/3.9 and you have a v
 
     $ pip install zappa
 
-Please note that Zappa _**must**_ be installed into your project's [virtual environment](http://docs.python-guide.org/en/latest/dev/virtualenvs/). The virtual environment name should not be the same as the Zappa project name, as this may cause errors.
+Please note that Zappa _**must**_ be installed into your project's [virtual environment](https://docs.python-guide.org/dev/virtualenvs/). The virtual environment name should not be the same as the Zappa project name, as this may cause errors.
 
-_(If you use [pyenv](https://github.com/yyuu/pyenv) and love to manage virtualenvs with **pyenv-virtualenv**, you just have to call `pyenv local [your_venv_name]` and it's ready. [Conda](http://conda.pydata.org/docs/) users should comment [here](https://github.com/Miserlou/Zappa/pull/108).)_
+_(If you use [pyenv](https://github.com/pyenv/pyenv) and love to manage virtualenvs with **pyenv-virtualenv**, you just have to call `pyenv local [your_venv_name]` and it's ready. [Conda](https://docs.conda.io/) users should comment [here](https://github.com/Miserlou/Zappa/pull/108).)_
```

Comment:
@matheusfelipeog Can you update `https://github.com/Miserlou/Zappa/pull/108` to `https://github.com/zappa/Zappa/issues/43`? We will see if we need to reopen it.

Reply:
Done @souravjamwal77